common/gcp-unified/src/main/java/ai/h2o/mojos/deploy/common/transform/unified/MojoPipelineToModelInfoConverter.java

```diff
@@ -0,0 +1,45 @@
+package ai.h2o.mojos.deploy.common.transform.unified;
```

Comment:
tbh none of this should be in the `common` project. It seems (even if lots of copy paste) that it is specific to gcp unified. So it deserves to go in gcp unified project I think. Otherwise, other projects will end up having this code packaged in due to build process.